app/src/main/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapper.kt

```diff
@@ -15,8 +14,7 @@ class BalanceMapper() {
 
     fun mapBalanceForWallet(balance: BigDecimal): String {
         return when {
-            balance.lessThenMinValue() -> BigDecimal.ZERO.asUsd()
-            balance < thousand -> balance.asUsd()
+            balance < thousand -> "$${balance.setScale(0, RoundingMode.FLOOR).stripTrailingZeros().toPlainString()}"
```

Comment:
can you extract the big balance.setScale(0, RoundingMode.FLOOR).stripTrailingZeros().toPlainString() to a val? it's a big string template